tests/e2e/serial/main_test.go

```diff
@@ -247,6 +247,89 @@ func TestSuiteScan(t *testing.T) {
 
 }
 
+func TestScanHasProfileGUID(t *testing.T) {
+	f := framework.Global
+	bindingName := framework.GetObjNameFromTest(t)
+	tpName := "test-scan-have-profile-guid-tp"
+	// This is the profileGUID for the redhat_openshift_container_platform_4.1 product and xccdf_org.ssgproject.content_profile_moderate profile
+	const profileGUIDOCPModerate = "d625badc-92a1-5438-afd7-19526c26b03c"
+	const profileGUIDTP = "d1359d86-c04f-5aa7-bcbc-e75a40844734"
+	const profileGUIDRHCOSModerate = "eceb9af0-17d4-5c59-9b17-07cfd22a3ba1"
+	const profileGUIDOCPCIS = "a230315d-3e4a-5b58-b00f-f96f1553e036"
+
+	f.AssertProfileGUIDMatches("ocp4-moderate", f.OperatorNamespace, profileGUIDOCPModerate)
```

Comment:
If this assertion fails, I don't think it will fail the test.

Most of our assertions return an error (using https://pkg.go.dev/fmt#Errorf). But, I think we need to check for that error and handle the failure.

If we were passing in the test suite, we could have the assertion call `t.Fatalf()`, but based on the current implementation we're relying on the test to enforce the outcome of the assertion (which I think we could argue that the assertion *should* handle failing the test, but that would be a broader change we would want to do consistently I think).

Reply:
That's a good catch!